enclave/core/hostcalls.c

```diff
@@ -13,6 +13,40 @@
 
 #include "core_t.h"
 
+#if !defined(OE_USE_BUILTIN_EDL)
+/**
+ * Declare the protoype of the following functions to avoid the
+ * missing-prototypes warning.
+ */
+oe_result_t _oe_log_ocall(uint32_t log_level, const char* message);
```

Comment:
It would be better to generate these prototypes via the `--header-only` edger8r option.
Otherwise these prototypes here can go out of sync with logging.edl

Reply:
This forward declaration is for the internal implemention so that we can make it weak alias with the public symbol generated by  oeeger8r. This means that the name of this internal implementation needs to be different than what oeeger8r generates if we use the same edl.